service/features/service.feature

```diff
@@ -942,3 +942,21 @@ Feature: VxFlex OS CSI interface
     When I call Probe
     And I call setQoSParameters with systemID "15dbbf5617523655" sdcID "d0f055a700000000" bandwidthLimit "10240" iopsLimit "10" volumeName "k8s-a031818af5" csiVolID "15dbbf5617523655-456ca4fc00000009" nodeID "9E56672F-2F4B-4A42-BFF4-88B6846FBFDA"
     Then the error contains "error setting QoS parameters"
+
```

Comment:
These look good, but I think we should add some negative test cases

Reply:
Negative test cases are added.